python/ray/data/_internal/execution/interfaces/physical_operator.py

```diff
@@ -128,8 +129,13 @@ def on_data_ready(self, max_bytes_to_read: Optional[int]) -> int:
         """
         bytes_read = 0
         while max_bytes_to_read is None or bytes_read < max_bytes_to_read:
+            block_ref, meta_ref = self._pending_block_pair or (
+                ray.ObjectRef.nil(),
+                ray.ObjectRef.nil(),
```

Comment:
Could we avoid using `ray.ObjectRef.nil()`? It's not a documented Ray Core API, and might break unexpectedly